react/hooks/useBulkImportDetailsQuery.ts

```diff
@@ -1,13 +1,28 @@
 import useSWR from 'swr'
 
 import { getBulkImportDetails } from '../services'
+import type { BulkImportDetails } from '../services/getBulkImportDetails'
 
-const useBulkImportDetailsQuery = (importId: string) => {
+export type UseBulkImportDetailsQueryProps = {
+  importId?: string
+  onSuccess?: (data: BulkImportDetails) => void
+  refreshInterval?: number
+}
+
+const useBulkImportDetailsQuery = ({
+  importId,
+  onSuccess,
+  refreshInterval = 0,
+}: UseBulkImportDetailsQueryProps) => {
   return useSWR(
     importId ? `/buyer-orgs/${importId}` : null,
     () => getBulkImportDetails(importId),
     {
+      refreshInterval,
       revalidateOnFocus: false,
+      onSuccess: data => {
```

Comment:
I think that you can just pass `onSuccess` here:

```suggestion
      onSuccess,
```